modules/enabling-scanner-v4-helm.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-helm_{context}"]
+= Enabling RHACS Scanner V4 when installing with Helm
+
+[role="_abstract"]
+Scanner V4 is not enabled by default, but you can enable it during installation.
+
+If you are installing {product-title-short} on secured clusters by using Helm charts, you must first enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. Create a `values-public.yaml` file that contains the following lines:
++
+[source,yaml]
+----
+scannerV4:
+    disable: false
+----
+. Run the command to install {product-title-short} with customizations, following the instructions in "Install Central using Helm charts with customizations":
++
+[source,terminal]
+----
+helm install -n stackrox --create-namespace \
+  stackrox-central-services rhacs/central-services \
+  -f <path_to_values_public.yaml> -f <path_to_values_private.yaml>
```

Comment:
Also mentioned in below comment, to keep the messaging for enabling / disabling scanner's consistent amongst the helm docs may be best to use CLI arguments (instead of a yaml file, or update all locations)

TL;DR; remove the step for `values-public.yaml` and instead add:

```suggestion
helm install -n stackrox --create-namespace \
  stackrox-central-services rhacs/central-services \
  -f <path_to_values_public.yaml> -f <path_to_values_private.yaml> \
  --set scannerV4.disable=false
```